cypress/integration/ambianic-tests/remote-connection.spec.js

```diff
@@ -17,10 +17,13 @@ context('RemoteConnections', () => {
     //     cy.get('#peerID').contains('5568ec87-42d8-47b0-aeea-01a125db0623')
     // })
 
-    it('Should switch to a remote Edge Peer ID', () => {
+    it('Should retrieve and display user PeerID', () => {
         // cy.get('#btn-settings').click()
         cy.get('#remotePeerID').type('917d5f0a-6469-4d33-b5c2-efd858118b74')
         cy.get('#btn-sendRemotePeerID').click()
-        cy.get('#edgePeerID').contains('917d5f0a-6469-4d33-b5c2-efd858118b74')
+
```

Comment:
@vickywane how about an assertion here that the PeerID is NOT visible before its revealed explicitly by clicking the toggle button?

Reply:
Okay, I would write a test for it.